apps/meeting/src/meetingConfig.ts

```diff
@@ -1,38 +1,32 @@
 // Copyright 2020-2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 // SPDX-License-Identifier: MIT-0
 
-import {
-  ConsoleLogger,
-  LogLevel,
-} from 'amazon-chime-sdk-js';
+import { ConsoleLogger, MultiLogger, POSTLogger } from 'amazon-chime-sdk-js';
 import { SDK_LOG_LEVELS } from './constants';
+import { MeetingConfig } from './types';
 
 const urlParams = new URLSearchParams(window.location.search);
-const queryLogLevel = urlParams.get('logLevel') || 'warn';
-const logLevel = SDK_LOG_LEVELS[queryLogLevel] || SDK_LOG_LEVELS.warn;
+const queryLogLevel = urlParams.get('logLevel') || 'info';
+const logLevel = SDK_LOG_LEVELS[queryLogLevel] || SDK_LOG_LEVELS.info;
 
-const BASE_URL: string = [
-  location.protocol,
-  '//',
-  location.host,
-  location.pathname.replace(/\/*$/, '/'),
-].join('');
-
-const postLogConfig = {
-  name: 'SDK_LOGS',
-  batchSize: 85,
-  intervalMs: 2000,
-  url: `${BASE_URL}logs`,
-  logLevel: SDK_LOG_LEVELS.info,
+const meetingConfig: MeetingConfig = {
+  simulcastEnabled: false,
+  logger: new ConsoleLogger('ChimeComponentLibraryReactDemo', logLevel),
 };
 
-export const logger = new ConsoleLogger('SDK', LogLevel.INFO);
+const BASE_URL: string = [location.protocol, '//', location.host, location.pathname.replace(/\/*$/, '/')].join('');
 
-const config = {
-  logLevel,
-  simulcastEnabled: false,
-  postLogConfig,
-  logger,
-};
+if (!['0.0.0.0', '127.0.0.1', 'localhost'].includes(location.hostname)) {
+  const postLogger = new POSTLogger({
+    url: `${BASE_URL}logs`,
+    logLevel,
+    metadata: {
+      appName: 'ChimeComponentLibraryReactDemo',
+      timestamp: Date.now().toString(), // Add current timestamp for unique AWS CloudWatch log stream generation. This will be unique per POSTLogger creation in time.
```

Comment:
unlikely but it is not unique if 2 ppl join at the exact same time right? Though our demo does not handle that case either so its probably fine but feel like just use uuid here is better?

Also could we use external meetingId and attendeeId so we could find the log easier?

Reply:
Will have to come back to this, but will track!